File: Task.Presentation/ClientApp/src/app/Employee.ts
export class Employee {
  id: number;
  name: string;
  hiringDate: Date;
  age: number;
  
}
export class EmployeeDTO {
  constructor(ID: number,
    Name: string,
    HiringDate: Date,
    Age: number) {
    this.ID = ID;
    this.Name = Name;
    this.HiringDate = HiringDate;
    this.Age = Age;
  }
  ID: number;
  Name: string;
  HiringDate: Date;
  Age: number;

}

File: Task.DAL/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace Task.DAL.Models
{
    public partial class Employee
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public int Age { get; set; }
        public DateTime HiringDate { get; set; }
   
    }
}
